src/main/java/frc/robot/subsystems/drive/SwerveModuleIOmk4ineo.java

```diff
@@ -0,0 +1,327 @@
+package frc.robot.subsystems.drive;
+
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix.sensors.CANCoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.ControlType;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
```

Comment:
Why is there so much commented out in this class? Remove the unnecessary commented out code